src/swirl.jl

```diff
@@ -0,0 +1,40 @@
+"""
+    swirl(img, rotation, strength, radius, x0 = OffsetArrays.center(img))
+
+Create whirlpool effect on `img`
+
+# Arguments
+- `img`: to specify input img array
+- `rotation`: to specify rotation angle
```

Comment:
It would be clearer to explain the rotation direction here; is it anti-clockwise or clockwise?